client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -39,7 +46,9 @@ class DailyBackupStatus extends Component {
 					{ translate( 'Latest backup completed:' ) }
 				</div>
 				<div className="daily-backup-status__date">{ displayDate }</div>
-				<Button className="daily-backup-status__download-button">Download backup</Button>
+				<Button className="daily-backup-status__download-button" onClick={ this.triggerDownload }>
+					Download backup
```

Comment:
Should this be translated already?

Reply:
Agree, at this point it's easy enough that we might as well go ahead and translate in initial PRs.